api/v1alpha1/etcd_types_test.go

```diff
@@ -91,12 +91,12 @@ func getEtcd(name, namespace string) *Etcd {
 	deltaSnapshotPeriod := metav1.Duration{
 		Duration: 300 * time.Second,
 	}
-	imageEtcd := "eu.gcr.io/gardener-project/gardener/etcd:v3.4.13-bootstrap"
-	imageBR := "eu.gcr.io/gardener-project/gardener/etcdbrctl:v0.12.0"
+	imageEtcd := "eu.gcr.io/gardener-project/gardener/etcd:v3.4.13-bootstrap-8"
+	imageBR := "eu.gcr.io/gardener-project/gardener/etcdbrctl:v0.20.2"
```

Comment:
Do we have to update this after every release backup-restore ?

Reply:
can we make this automated like it can read the image tag from version may be ?

Reply:
I think this will then be part of `make generate`. And since this is just an API test, the values don't even matter. We can use dummy values here if necessary, like the value for `container` field.